docusaurus.config.js

```diff
@@ -131,6 +131,11 @@ const config = {
         disableSwitch: true,
         respectPrefersColorScheme: true,
       },
+      metadata: [
+        { property: 'og:description', content: 'Developer Docs for passwordless authentication' },
+        { property: 'og:image', content: '/assets/zero-trust-passwords.webp' },
```

Comment:
This may need to be an absolute url. [Stack Overflow](https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url)